game_eggs/steamcmd_servers/palworld/egg-palworld.json

```diff
@@ -0,0 +1,94 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-01-19T19:39:44+01:00",
+    "name": "Palworld",
+    "author": "admin@ballaual.de",
+    "description": "Fight, farm, build and work alongside mysterious creatures called \"Pals\" in this completely new multiplayer, open world survival and crafting game!",
+    "features": null,
+    "docker_images": {
+        "ghcr.io\/parkervcp\/steamcmd:debian": "ghcr.io\/parkervcp\/steamcmd:debian"
+    },
+    "file_denylist": [
+        "PalServer.sh"
+    ],
+    "startup": "\/home\/container\/Pal\/Binaries\/Linux\/PalServer-Linux-Test Pal  -port={{SERVER_PORT}} -players={{MAX_PLAYERS}} -useperfthreads -NoAsyncLoadingThread -UseMultithreadForDS -publicip {{SERVER_IP}} -publicport {{SERVER_PORT}}   EpicApp=PalServer -servername=\"{{SRV_NAME}}\" -serverpassword=\"{{SRV_PASSWORD}}\" -adminpassword=\"{{ADMIN_PASSWORD}}\"",
```

Comment:
Does there need to be a `-` before the EpicApp or anything there?

Reply:
Not according to the documentation, i removed the extra spaces because it annoyed me.

The dedicated server browser in game is currently bugged so I havent been able to verify if it even shows up. Direct connect works though, currently have 4 friends playing on my server

Reply:
> Does there need to be a `-` before the EpicApp or anything there?

![image](https://github.com/parkervcp/eggs/assets/38478976/1ca8dbc0-8209-410b-918a-6c26fe8f5693)

according to the official docu no

Reply:
> The dedicated server browser in game is currently bugged so I havent been able to verify if it even shows up.

you might need to spam the button like 50-100 times to make it load enough servers where your server is located 🤮